### ISSUE TITLE
`sway-fmt` formats match enum variants into one long line

### ISSUE THREAD

**Issue by Braqzen** [FuelLabs/sway#1689]
Related: #1493 #1210 

The following

```
contract;

use std::{chain::auth::Sender, contract_id::ContractId, token::{force_transfer, transfer_to_output}};

abi MyContract {
    fn test_function(to: Sender, asset_id: ContractId, value: u64) -> bool;
}

impl MyContract for Contract {
    fn test_function(to: Sender, asset_id: ContractId, value: u64) -> bool {
        match to {
            Sender::Address(address) => transfer_to_output(value, asset_id, address), 
            Sender::ContractId(contract) => force_transfer(value, asset_id, contract), 
        };

        true
    }
}
```

when running `forc fmt` (version 0.13.2) formats to 
```
contract;

use std::{chain::auth::Sender, contract_id::ContractId, token::{force_transfer, transfer_to_output}};

abi MyContract {
    fn test_function(to: Sender, asset_id: ContractId, value: u64) -> bool;
}

impl MyContract for Contract {
    fn test_function(to: Sender, asset_id: ContractId, value: u64) -> bool {
        match to {
            Sender::Address(address) => transfer_to_output(value, asset_id, address), Sender::ContractId(contract) => force_transfer(value, asset_id, contract), 
        };

        true
    }
}
```

**Comment by eureka-cpu** [comment]
This should be closed by #1873 once we switch to the new formatter

**Comment by eureka-cpu** [comment]
Closed by #2539
